src/test/java/testsClasses/JsonParserTest.java

```diff
@@ -0,0 +1,76 @@
+package testsClasses;
+
+import dataProviders.DataProviderForTest;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import parser.JsonParser;
+import parser.Parser;
+import shop.Cart;
+
+import java.io.File;
+
+public class JsonParserTest {
+
+    private Parser jsonParser;
+    private Cart cart;
+    private File file;
+
+    @BeforeClass(alwaysRun = true)
+    public void setUpBeforeClass() {
+        jsonParser = new JsonParser();
+    }
+
+    @AfterMethod(alwaysRun = true)
+    public void cleanResources() {
+        file = new File("src/main/resources/");
+        for (File item : file.listFiles()) {
+            if (!(item.getName().equals("andrew-cart.json") | item.getName().equals("eugen-cart.json"))) {
+                item.delete();
+            }
+        }
+    }
+
+    @Test(groups = {"smoke", "positive", "allFunctions"}, dataProvider = "cartNamesPositive", dataProviderClass = DataProviderForTest.class)
+    public void writeToFileIsFilePresentPositive(String cartName) {
+        String fileName = cartName + ".json";
+        cart = new Cart(cartName);
+
+        jsonParser.writeToFile(cart);
+
+        Assert.assertTrue(isFilePresent(fileName));
```

Comment:
Add second parameter (message on failure) for assertTrue/assertFalse, otherwise if test fails then you will see message "Expected true, but found false".